scripts/generate-clients/config.js

```diff
@@ -1,5 +1,5 @@
 // Update this commit when taking up new changes from smithy-typescript.
 module.exports = {
   // Use full commit hash as we explicitly fetch it.
-  SMITHY_TS_COMMIT: "93274e37a28e508e5137493ffdf7aa63b0c8daed",
+  SMITHY_TS_COMMIT: "6db52d8fd544697a80a50d54f4731aae9eec40a9",
```

Comment:
TODO: This should be updated to the smithy-typescript main commit of https://github.com/awslabs/smithy-typescript/pull/1060

Marking my review as requesting changes until this is done